### DISCUSSION TITLE
Account(s) Locked!

### DISCUSSION THREAD

**Discussion by VictimaVitae** [charlesbel/Microsoft-Rewards-Farmer#370] [General]
Some of my accounts were locked by Microsoft for breaking their terms of service due to suspicious activity. 

Edit: I was able to unlock my accounts using my phone number. However, I do not know if I will be on their watchlist. Microsoft are low.

**Comment by Vaanaattori** [comment]
Can you update on if Microsoft has detected your accounts?

**Reply by VictimaVitae** [reply]
@Vaanaattori they did not detect my accounts, they only detected one of my accounts when I tried to redeem two awards in one day. But I haven't used the bot in a while.
